tools/mkdeps.c

```diff
@@ -443,12 +446,13 @@ static void parse_args(int argc, char **argv)
            * do_dependency() uses them as bare filenames as well.
            * (In addition to passing them to system().)
            */
-
+#ifndef CONFIG_WINDOWS_NATIVE
           if (group == 1)
             {
                arg = do_shquote(arg);
```

Comment:
what's error here?

Reply:
`'-fno-common': No such file or directory sparc-elf-gcc: '-Wall': No such file or directory sparc-elf-gcc: '-Wstrict-prototypes': No such file or directory sparc-elf-gcc: '-Wshadow': No such file or directory sparc-elf-gcc: '-Wundef': No such file or directory sparc-elf-gcc: '-fomit-frame-pointer': No such file or directory sparc-elf-gcc: '-g': No such file or directory sparc-elf-gcc: '-mcpu=leon': No such file or directory sparc-elf-gcc: '-isystem': No such file or directory`

Reply:
do_shquote just add ", it's strange that it can make this error

Reply:
it seems that in Windows native build gcc can only  identification option like this:
`xxxxxxx-gcc.exe -fno-common: -Wall -Wstrict-prototypes -Wshadow`
rather than:
`xxxxxxx-gcc.exe '-fno-common '-Wall '-Wstrict-prototypes' '-Wshadow'`

Reply:
If so, let's remove quote for linux too?

Reply:
I'm not sure